app/views/pages/closed.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+    <div class="row pl-2 mt-4">
+        <h1>Closed</h1>
+    </div>
+    <div class="row pl-2">
+        <p>Unfortunately the deadline for Life of the Mind has been reached, and is no longer accepting any submissions. If you have any questions, please contact Melissa Cox Norris at melissa.norris@uc.edu or by phone at (513) 556-1558.</p>
```

Comment:
I provided the text that Melissa wants used in a comment for this issue.  Please use that instead.

Also, the header should be "Artists, Authors, Editors & Composers" instead of "Closed".  We want people to know they've reached the correct form.